pgdrive/scene_manager/spawn_manager.py

```diff
@@ -35,6 +38,9 @@ def __init__(self, exit_length, lane_num, num_agents, vehicle_config, target_veh
         self.initialized = False
         self.target_vehicle_configs = target_vehicle_configs
         self.spawn_places_used = []
+        self.vehicle_length = vehicle_config["vehicle_length"]
+        self.vehicle_width = vehicle_config["vehicle_width"]
+        self.custom_target_vehicle_config = True if target_vehicle_configs is not None else False
```

Comment:
Why we hard coded the vehicle length here? Can we remove it and query the config everytime?

Reply:
I have to get this value before vehicle created to decide the spawn points